src/commons/mocks/BackendMocks.ts

```diff
@@ -181,9 +181,24 @@ export function* mockBackendSaga(): SagaIterator {
     FETCH_GRADING_OVERVIEWS,
     function* (action: ReturnType<typeof actions.fetchGradingOverviews>): any {
       const accessToken = yield select((state: OverallState) => state.session.accessToken);
-      const { filterToGroup, pageParams, filterParams } = action.payload;
+      const { filterToGroup, pageParams, filterParams, allColsSortStates } = action.payload;
+      const sortedBy = {
+        sortBy: allColsSortStates.sortBy,
+        sortDirection: ''
+      };
+      for (const key in allColsSortStates.currentState) {
```

Comment:
Is this type safe? How is it different from `Object.keys(allColsSortStates.currentState).forEach(...)`?